package.json.unit.test.js

```diff
@@ -11,8 +11,20 @@ describe('because rollup 0.60 drops support for Internet Explorer 8', () => {
   describe('rollup', () => {
     it('should be pinned to 0.56.5 in package-lock.json', () => {
       expect(packageLockJson.dependencies.rollup.version).toEqual('0.56.5')
-      expect(packageLockJson.dependencies['gulp-better-rollup'].dependencies).toBeUndefined()
-      expect(packageLockJson.dependencies['gulp-better-rollup'].requires.rollup).toEqual('>=0.48 <0.57')
+
+      // Rollup pinned
+      expect(packageLockJson.dependencies['gulp-better-rollup'])
+        .toEqual(expect.objectContaining({
+          requires: expect.objectContaining({
+            rollup: '>=0.48 <0.57'
+          })
+        }))
+
+      // Rollup has been hoisted (no longer a child dependency)
+      expect(packageLockJson.dependencies['gulp-better-rollup'])
+        .not.toEqual(expect.objectContaining({
+          dependencies: expect.any(Object)
+        }))
```

Comment:
I fixed everything major that the TypeScript compiler flagged up

The old test crashes out because it literally knows the **package.json** child `dependencies` field literally doesn't exist 🙈 

So now it looks at the object shape `not.toEqual()`

As in, doesn't equal an object with a `dependencies` property